src/components/Settings/People/People.component.js

```diff
@@ -95,126 +90,125 @@ const People = ({
         disableSubmit={!isLogged}
       >
         <Paper>
-          <List>
-            <ListItem>
-              <div className="Settings__UserIcon__Container">
-                <UserIcon />
-              </div>
-              <ListItemText primary={name} />
-              <ListItemSecondaryAction className="Settings--secondaryAction">
+          <Box px={3} py={4}>
+            {/* Icono y Logout */}
+            <Box
+              display="flex"
+              flexDirection="column"
+              alignItems="center"
+              mb={4}
+            >
+              <UserIcon />
+              <Typography variant="h6" sx={{ mt: 2, textAlign: 'center' }}>
+                {name}
+              </Typography>
+              <Box display="flex" justifyContent="center" width="100%">
```

Comment:
This box is not necessary because it only have a Button as child and we can resolve this styles with the parent box